docs/API.md

```diff
@@ -242,6 +249,24 @@ The logging level is a *minimum* level. For instance if `logger.level` is
 
 You can pass `'silent'` to disable logging.
 
+It is possible to pass an object to set different levels to the logger and its
+children. The syntax for designating the target loggers is borrowed from the
+[debug](https://github.com/visionmedia/debug#wildcards) module's `DEBUG` variable
+(also supported in [pino-debug](https://github.com/pinojs/pino-debug)):
++ use `'*'`(star) as wildcard (rules are ordered so that `'*'` will not override
+`'mylib*'`).
```

Comment:
No, I do not want people to think that they need `debug` to use that feature (or to use `pino` for that matter), but felt obliged to credit that project. But you are right it is confusing, I will rephrase to just describe the syntax and add a parenthesis `(inspired by the debug and pino-debug modules)`